Update faq.md to reflect that `tmux-resurrect` already prunes the backup directory.

lexjacobs opened the pull request:
This command has been tested on osx mojave running tmux on iTerm2 with zsh.
The delete command was taken from: https://stackoverflow.com/a/34862475/3044358

bruno- commented:
How does this feature relate to this one that's already built into tmux-resurrect

https://github.com/tmux-plugins/tmux-resurrect/blob/da1a7558024b8552f7262b39ed22e3d679304f99/scripts/save.sh#L271-L277

lexjacobs commented:
Oh! Great. Thanks for pointing this out.
I updated the FAQ page to reflect this info. 
In its current state, it suggests you have to manually prune the files in the `resurrect` directory.

Updated PR title and contents

bruno- commented:
Thanks
